packages/calcite-components/src/components/table-cell/table-cell.tsx

```diff
@@ -61,6 +61,9 @@ export class TableCell
   /** @internal */
   @Prop() lastCell: boolean;
 
+  /** @internal */
+  @Prop() nonInteractive = false;
```

Comment:
How about something like `interaction-disabled`? This would align better with how we disable features throughout components. Our props should be in the tense as `disabled`.

Reply:
Yeah, that's a better boolean name.

Any preference between `interactionDisabled` and `interactionMode="non-interactive" | "interactive"` (or another property / value option)?

Reply:
we have `selectionMode` on a few other components but i'm not sure its the same thing.

Reply:
Yeah, different functionality, Table has `selectionMode` as well. Could see the similarity as good or bad. Either way, `interactionDisabled` is better than `nonInteractive`. cc @jcfranco for any thoughts.

Reply:
what about `interactionMode="static" | "interactive"`

Reply:
Yeah, that makes sense to me as well.

Reply:
I've updated the PR to use interactionMode="static" | "interactive"

Reply:
I like @alisonailea's suggestion. 👍

Reply:
Cool, I think this one should be good to go then. I'll re-submit for approval.